Default the health endpoint port to 8083

mprahl opened the pull request:
This makes it not conflict with the spec sync health endpoint by
default.

Related:
https://github.com/open-cluster-management-io/governance-policy-addon-controller/commit/918eeb7739f8db71f386068ac7eb236aac2c3859
https://github.com/stolostron/backlog/issues/24148

Signed-off-by: mprahl <mprahl@users.noreply.github.com>

mprahl committed:
Default the health endpoint port to 8083

This makes it not conflict with the spec sync health endpoint by
default.

Related:
https://github.com/open-cluster-management-io/governance-policy-addon-controller/commit/918eeb7739f8db71f386068ac7eb236aac2c3859
https://github.com/stolostron/backlog/issues/24148

Signed-off-by: mprahl <mprahl@users.noreply.github.com>

mprahl commented:
@JustinKuli I know this isn't required because you override the default in the policy addon controller but this makes it match with the other controllers in the policy framework.

JustinKuli commented:
Ah, you're right, I thought that status-sync was defaulting to :8081, and we only overrode it in the actual deployment. Good catch!

dhaiducek commented:
@mprahl Looks like `template-sync` is pulling in the new object deletion CRD and the E2E test is failing.

mprahl committed:
Fix the e2e tests

A new default field of pruneObjectBehavior was recently added on the
ConfigurationPolicy CRD and the tests need to account for that.

Signed-off-by: mprahl <mprahl@users.noreply.github.com>

mprahl commented:
Thanks for the heads-up @dhaiducek! I just made the test change.

dhaiducek reviewed (approved):
Thanks, @mprahl!
